### ISSUE TITLE
Article titles in italics

### ISSUE THREAD

**Issue by desb42** [gnosygnu/xowa#277]
While pursuing another issue #261 I looked at the template infobox_opera
![noitalics](https://user-images.githubusercontent.com/861631/48399259-13f8b600-e71b-11e8-8e74-0d0a6289a733.jpg)
And on enwiki
![italics](https://user-images.githubusercontent.com/861631/48399413-76ea4d00-e71b-11e8-83d4-e58552b5c9b8.jpg)

The difference is that the article title in one is in italics the other is not.
I tried to look at this, but got bogged down in the Lua code
I know that this is a very picky in an obscure area but ...

P.S.
I have just noticed that the padlock icons on the right are of different colours.
Is this Mediawiki doing further 'tweeks'?

**Comment by desb42** [comment]
The same change to L36 of xowa/400_xowa/src/gplx/xowa/xtns/pfuncs/pages/Pfunc_displaytitle.java
also works

I agree the protection scheme is far too complicated for a desktop or small numbers of user webserver.

**Comment by gnosygnu** [comment]
> The same change to L36 of xowa/400_xowa/src/gplx/xowa/xtns/pfuncs/pages/Pfunc_displaytitle.java
also works

Oops. I didn't realize I linked to xowa_android above instead of xowa.

Still, I think it should be L39 instead of L36? https://github.com/gnosygnu/xowa/blob/master/400_xowa/src/gplx/xowa/xtns/pfuncs/pages/Pfunc_displaytitle.java#L39

Let me know if I pushed anything wrong just now

> I agree the protection scheme is far too complicated for a desktop or small numbers of user webserver.

Yeah, I don't see a compelling use-case for this. I might re-evaluate if web-server ever becomes more prevalent in a multi-user setting, but otherwise, just planning to close this issue.

Thanks!
